scripts/pin_the_slow_way.py

```diff
@@ -39,10 +39,12 @@
           'libblitz': 'libblitz 0.10|0.10.*',
           'libmatio': 'libmatio 1.5.*',
           'libnetcdf': 'libnetcdf 4.4.*',
-          'libpng': 'libpng >=1.6.21,<1.7',
+          'libpng': 'libpng >=1.6.23,<1.7',
```

Comment:
This is more restrictive (an working in some cases) than the current setup. So I am merging this to get the other changes in too.  We have another PR that pins `libpng` to the bone and any further discussion should go there.

See https://github.com/conda-forge/conda-forge.github.io/pull/211